NEWS.md

```diff
@@ -35,6 +35,13 @@ Language changes
   * Nested `if` expressions that arise from the keyword `elseif` now use `elseif`
     as their expression head instead of `if` ([#21774]).
 
+  * Parsed and lowered forms of type definitions have been synchronized with their
+    new keywords ([#23157]). Expression heads are renamed as follows:
+    `type`           => `struct`
```

Comment:
Perhaps add `*`s on this list with 4 space indent to render it as a bullet list? Current rendering looks a bit funny https://github.com/JuliaLang/julia/blob/acadf3acb6054adb657d6a9390616f2c1e792546/NEWS.md#language-changes